### PR TITLE
Rename APIs that have been reported for causing rejection :(

### PR DESCRIPTION
**Pull request by garrettmoon**
This is dumb, but if it means fewer rejections for clients it's probably the right call.

### CODE REVIEW

File: Pod/Classes/PINAnimatedImage.m [hunk 1]
```diff
@@ -127,7 +127,7 @@ void releaseData(void *data, const void *imageData, size_t size)
   CFRelease(data);
 }
 
-- (CGImageRef)imageAtIndex:(NSUInteger)index inSharedImageFiles:(NSArray <PINSharedAnimatedImageFile *>*)imageFiles width:(UInt32)width height:(UInt32)height bitsPerPixel:(UInt32)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
+- (CGImageRef)imageForIndex:(NSUInteger)index inSharedImageFiles:(NSArray <PINSharedAnimatedImageFile *>*)imageFiles width:(UInt32)width height:(UInt32)height bitsPerPixel:(UInt32)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
```
Comment (maicki) [inline Pod/Classes/PINAnimatedImage.m]: Curious why did you change it to `imageForIndex:...`? `objectAtIndex:...` is I think the common way in Apple APIs

File: Pod/Classes/PINAnimatedImage.m [hunk 2]
```diff
@@ -155,12 +155,12 @@ - (CGImageRef)imageAtIndex:(NSUInteger)index inSharedImageFiles:(NSArray <PINSha
   return nil;
 }
 
-- (CFTimeInterval)durationAtIndex:(NSUInteger)index
+- (CFTimeInterval)durationForIndex:(NSUInteger)index
```
Comment (maicki) [inline Pod/Classes/PINAnimatedImage.m]: Same as my first comment change from `At` to `For`

File: Pod/Classes/PINAnimatedImage.m [hunk 3]
```diff
@@ -391,24 +391,24 @@ @implementation PINSharedAnimatedImage
...
+    _firstImageLock = [[PINRemoteLock alloc] initWithName:@"PINSharedAnimatedImage cover image lock"];
     _completions = @[];
     _infoCompletions = @[];
     _maps = @[];
   }
   return self;
 }
 
-- (void)setInfoProcessedWithCoverImage:(PINImage *)coverImage UUID:(NSUUID *)UUID durations:(Float32 *)durations totalDuration:(CFTimeInterval)totalDuration loopCount:(size_t)loopCount frameCount:(size_t)frameCount width:(size_t)width height:(size_t)height bitsPerPixel:(size_t)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
+- (void)setInfoProcessedWithFirstImage:(PINImage *)firstImage UUID:(NSUUID *)UUID durations:(Float32 *)durations combinedDurations:(CFTimeInterval)combinedDurations loopCount:(size_t)loopCount frameCount:(size_t)frameCount width:(size_t)width height:(size_t)height bitsPerPixel:(size_t)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
```
Comment (maicki) [inline Pod/Classes/PINAnimatedImage.m]: What about changing that to `setInforProcessedForFirstImage:...`

File: Pod/Classes/PINAnimatedImage.m [hunk 4]
```diff
@@ -155,12 +155,12 @@ - (CGImageRef)imageAtIndex:(NSUInteger)index inSharedImageFiles:(NSArray <PINSha
...
 }
 
-- (CFTimeInterval)durationAtIndex:(NSUInteger)index
+- (CFTimeInterval)durationForIndex:(NSUInteger)index
 {
   return self.durations[index];
 }
 
-+ (CGImageRef)imageAtIndex:(NSUInteger)index inMemoryMap:(NSData *)memoryMap width:(UInt32)width height:(UInt32)height bitsPerPixel:(UInt32)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
++ (CGImageRef)imageForIndex:(NSUInteger)index inMemoryMap:(NSData *)memoryMap width:(UInt32)width height:(UInt32)height bitsPerPixel:(UInt32)bitsPerPixel bitmapInfo:(CGBitmapInfo)bitmapInfo
```
Comment (maicki) [inline Pod/Classes/PINAnimatedImage.m]: Same as my first comment change from `At` to `For`

**Comment by garrettmoon** [comment]
I think in the end this is the wrong direction.
